plugins/org.eclipse.fordiac.ide.export.forte_ng/src/org/eclipse/fordiac/ide/export/forte_ng/base/BaseFBHeaderTemplate.xtend

```diff
@@ -141,4 +129,11 @@ abstract class BaseFBHeaderTemplate<T extends BaseFBType> extends ForteFBTemplat
 			methodLanguageSupport.values.filterNull.flatMap[getDependencies(options)] + type.internalFbs.map[getType]
 			).toSet
 	}
+	
+	def private generateInternalFBDeclarations() '''
+		«FOR fb : type.internalFbs»
+			«fb.type.generateTypeName» «FB_EXPORT_PREFIX»«fb.name»;
```

Comment:
Please use `fb.generateName` instead of `«FB_EXPORT_PREFIX»«fb.name»`.

Reply:
I didn't feel to comfortable using `generateName` as in the previouse version it added the `()` and therefore I was not sure if this is the intended use case.

Changed now.